### ISSUE TITLE
Version bump for Ollama?

### ISSUE THREAD

**Issue by davidrpugh** [conda-forge/ollama-feedstock#20]
### Comment:

@sodre thanks for creating this feedstock for Ollama! The current version of Ollama is 0.3.10 and the current version of Ollama provided by this feedstock is 0.1.17.

How can I help update this feedstock to keep it updated when newer versions of Ollama are released?

**Comment by sanjanacheerla** [comment]
Yes please, can we get an update with the latest Ollama version!

**Comment by benmoss** [comment]
we're up to date again with 0.6.2!
